pkg/auth/login_credentials_manager_test.go

```diff
@@ -302,76 +220,24 @@ func (suite *LoginCredentialsManagerTestSuite) TestGetConfluentPrerunCredentials
 	suite.compareCredentials(envPrerunCredentialsWithCaCertPath, creds)
 }
 
-func (suite *LoginCredentialsManagerTestSuite) TestGetConfluentPrerunCredentialsFromNetrc() {
-	var params netrc.NetrcMachineParams
-
-	// no cacertpath
-	netrcHandler := &mock.NetrcHandler{
-		GetMatchingNetrcMachineFunc: func(_ netrc.NetrcMachineParams) (*netrc.Machine, error) {
-			return confluentNetrcPrerunMachine, nil
-		},
-		GetFileNameFunc: func() string {
-			return netrcFileName
-		},
-	}
-	loginCredentialsManager := NewLoginCredentialsManager(netrcHandler, suite.prompt, suite.ccloudClient)
-	creds, err := loginCredentialsManager.GetOnPremPrerunCredentialsFromNetrc(params)()
-	suite.require.NoError(err)
-	suite.compareCredentials(netrcPrerunCredentials, creds)
-
-	// with cacertpath
-	netrcHandler = &mock.NetrcHandler{
-		GetMatchingNetrcMachineFunc: func(_ netrc.NetrcMachineParams) (*netrc.Machine, error) {
-			return confluentNetrcPrerunMachineWithCaCertPath, nil
-		},
-		GetFileNameFunc: func() string {
-			return netrcFileName
-		},
-	}
-	loginCredentialsManager = NewLoginCredentialsManager(netrcHandler, suite.prompt, suite.ccloudClient)
-	creds, err = loginCredentialsManager.GetOnPremPrerunCredentialsFromNetrc(params)()
-	suite.require.NoError(err)
-	suite.compareCredentials(netrcPrerunCredentialsWithCaCertPath, creds)
-}
-
 func (suite *LoginCredentialsManagerTestSuite) TestGetCredentialsFunction() {
-	noCredentialsNetrcHandler := &mock.NetrcHandler{
-		GetMatchingNetrcMachineFunc: func(params netrc.NetrcMachineParams) (*netrc.Machine, error) {
-			return nil, nil
-		},
-		GetFileNameFunc: func() string {
-			return netrcFileName
-		},
-	}
 	cmd := &cobra.Command{}
 	cmd.Flags().Bool("save", false, "test")
-	// No credentials in env var and netrc so should look for prompt
-	loginCredentialsManager := NewLoginCredentialsManager(noCredentialsNetrcHandler, suite.prompt, suite.ccloudClient)
+	// No credentials in env var so should look for prompt
```

Comment:
Unrelated, but can we specify which "env var" we're talking about in this comment?